### ISSUE TITLE
Forestry 3.5.2.10 Crash

### ISSUE THREAD

**Issue by Zac666** [ForestryMC/ForestryMC#597]
Crash happens when I was digging, but on base I have chunkloaded multifarm, now world crash on load, crash reports same

https://gist.github.com/mezz/19797b5db731c7fad904

**Comment by UltraPeeks** [comment]
I just got the same crash. It now happens everytime I try to load the world.
http://pastebin.com/uH8dcXsP
http://openeye.openmods.info/crashes/b492171d4d119263329e236d24056ac8

**Comment by leagris** [comment]
Just had same crash with 3.6.0.520
```java
java.lang.NullPointerException: Ticking entity
        at forestry.core.genetics.Genome.getActiveAllele(Genome.java:162)
        at forestry.lepidopterology.genetics.ButterflyGenome.getFlowerProvider(B
utterflyGenome.java:117)
        at forestry.lepidopterology.entities.AIButterflyPollinate.canInteract(AI
ButterflyPollinate.java:39)
        at forestry.lepidopterology.entities.AIButterflyInteract.func_75250_a(AI
ButterflyInteract.java:40)
        at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:70)
        at net.minecraft.entity.EntityLiving.func_70619_bc(EntityLiving.java:540
)
        at forestry.lepidopterology.entities.EntityButterfly.func_70619_bc(Entit
yButterfly.java:477)
        at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.j
ava:1772)
        at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:367)
        at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.
java:1609)
        at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:206)
        at forestry.lepidopterology.entities.EntityButterfly.func_70071_h_(EntityButterfly.java:436)
        at net.minecraft.world.World.func_72866_a(World.java:2070)
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:648)
        at net.minecraft.world.World.func_72870_g(World.java:2034)
        at net.minecraft.world.World.func_72939_s(World.java:1887)
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
        at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
-- Entity being ticked --
Details:
        Entity Type: Forestry.butterflyGE (forestry.lepidopterology.entities.EntityButterfly)
        Entity ID: 154404
        Entity Name: entity.Forestry.butterflyGE.name
        Entity's Exact location: -106,60, 79,00, 233,55
        Entity's Block location: World: (-107,79,233), Chunk: (at 5,4,9 in -7,14; contains blocks -112,0,224 to -97,255,239), Region: (-1,0; contains chunks -32,0 to -1,31, blocks -512,0,0 to -1,255,511)
        Entity's Momentum: -0,03, -0,09, 0,02
```

```
        Minecraft Version: 1.7.10
        Operating System: Linux (amd64) version 3.16.0-33-generic
        Java Version: 1.7.0_76, Oracle Corporation
        Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
        Memory: 752231320 bytes (717 MB) / 1725014016 bytes (1645 MB) up to 4080271360 bytes (3891 MB)
        JVM Flags: 17 total; -Xms256M -Xmx4G -XX:MaxPermSize=256M -XX:NewRatio=3 -XX:SurvivorRatio=3 -XX:TargetSurvivorRatio=80 -XX:MaxTenuringThreshold=8 -XX:+UseParNewGC -XX:+UseConcMarkSweepGC -XX:MaxGCPauseMillis=10 -XX:GCPauseIntervalMillis=50 -XX:MaxGCMinorPauseMillis=7 -XX:+ExplicitGCInvokesConcurrent -XX:+UseCMSInitiatingOccupancyOnly -XX:CMSInitiatingOccupancyFraction=60 -XX:+BindGCTaskThreadsToCPUs -Xnoclassgc
```

**Comment by mezz** [comment]
Thanks for the report!
I took down 3.5.2 for now. I'll get to this soon.

**Comment by mezz** [comment]
Should be fixed. Can you please try out the latest dev build to confirm that it's fixed for you?
http://jenkins.ic2.player.to/job/Forestry_Dev/

**Comment by Zac666** [comment]
I'm using builds from http://jenkins.ic2.player.to/job/Forestry/
And with latest build everythink works fine, thank you very much for your work ^_^

**Comment by mezz** [comment]
Cool, fix has been added to Stable 3.5.3
